test/fixtures.test.ts

```diff
@@ -47,6 +47,18 @@ describe.concurrent('fixtures', () => {
     expect(js).contains('uno-tacwqa')
   })
 
+  it('vite legacy', async () => {
+    const root = resolve(__dirname, 'fixtures/vite-legacy')
+    await fs.emptyDir(join(root, 'dist'))
+    await execa('npm', ['run', 'build'], { cwd: root })
```

Comment:
### Additional Information
I try using `await build({...})` first. but the test case not work and `../src/uno.svg referenced in assets/index-!~{001}~.js-unocss-hash.css didn't resolve at build time, it will remain unchanged to be resolved at runtime` error appears

And then i use `await execa('npm', ['run', 'build'], { cwd: root })` instead of it just work fine.

Reply:
fixes: https://github.com/unocss/unocss/pull/2571/commits/dac2bd5acfaf524a8341938c994c9262721002a8